### DISCUSSION TITLE
Super table Chrome Crashing bug

### DISCUSSION THREAD

**Discussion by Mozix** [Budibase/budibase#12487] [Help] [answered]
Hello,

I have discovered an issue with super table plug-in.

When using super table and it's super column as well as super cell components, if you click on the column title Chrome starts eating your RAM and eventually the site freezes and you get the site is not responding alert.

**Comment by ConorWebb96** [comment, marked as answer]
Hey @Mozix,

It would be better to create an issue on the repo for the plugin as we don't manage/nor have created that plugin. You should be able to do that [here](https://github.com/poirazis/bb-component-SuperTable/issues).

I hope this makes sense.
